strax/processing/peak_properties.py

```diff
@@ -68,12 +68,18 @@ def compute_index_of_fraction(peak, fractions_desired, result):
 
 
 @export
-def compute_widths(peaks):
+def compute_widths(peaks, select_peaks_indices=None):
     """Compute widths in ns at desired area fractions for peaks
```

Comment:
could you update the dosctring to include the select_peaks_indices and what None means?